pywbem/mof_compiler.py

```diff
@@ -1897,8 +1897,15 @@ def DeleteQualifier(self, *args, **kwargs):
 class MOFWBEMConnection(BaseRepositoryConnection):
     """
     A CIM repository connection to an in-memory repository on top of an
-    underlying repository, that is used by the MOF compiler to provide rollback
-    support.
+    underlying repository API, that is used by the MOF compiler to provide
+    rollback support.  It only implementats the BaseRepositoryConnection
```

Comment:
I think with gaining more clarity about the nature of this construct, I suggest changing the first sentence to:

"A CIM repository connection to an in-memory store on top of an optional underlying WBEM connection.  If the underlying WBEM connection is provided, the connection is the target for any operations and the in-memory store acts as a rollback log to provide support for rolling back the operations. If the underlying WBEM connection is not provided, the in-memory store acts as a CIM repository that is targeted by the operations."